webui/react/src/components/Transfer.tsx

```diff
@@ -166,19 +159,19 @@ const Transfer: React.FC<Props> = ({
   );
 
   const renderHiddenRow = useCallback(
-    ({ index, style }: { index: number; style: React.CSSProperties }) => {
+    ({ index }: { index: number }) => {
```

Comment:
nit: you can simplify this line (and the one in `renderVisibleRow`) to just `(index: number) => {`